respown_point: [[6.0, 0.0, 90.0],
                [6.0, 4.0, 90.0], 
                [4.5, 7.0 ,135.0],
                [0.0, 8.0, 180],
                [-4.5, 7.0, -135.0],
                [-6.0, 4.0, -90.0],
                [-6.0, 0.0, -90.0],
                [-6.0, -4.0, -90.0], 
                [-4.5, -7.0 ,-45.0],
                [0.0, -8.0, 0.0],
                [4.5, -7.0, 45.0],
                [6.0, -4.0, 90.0],
                ]